src/common/droid-util.c

```diff
@@ -91,6 +91,8 @@ struct droid_option valid_options[] = {
     { "speaker_before_voice",   DM_OPTION_SPEAKER_BEFORE_VOICE  },
     { "output_voip_rx",         DM_OPTION_OUTPUT_VOIP_RX        },
     { "record_voice_16k",       DM_OPTION_RECORD_VOICE_16K      },
+    { "halium_10_audio_hal",    DM_OPTION_HALIUM_10_AUDIO_HAL   },
```

Comment:
I think the current naming is too generic?

how about something like `use_legacy_stream_set_parameters`?